Samsung/WATT
# Update Design Editor to 0.4.1

[Issue] N/A
[Problem] DE can not open TAU samples which are downloaded
    from TAU GitHub Page
[Solution] Bump Design Editor
[TEST]
    1. Open http://localhost:3000/demos/?path=mobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fcheckbox.html
    2. Open Design Editor
    3. Design Editor and its Live preview should be working.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>
